recipes/orator/meta.yaml

```diff
@@ -0,0 +1,87 @@
+{% set name = "orator" %}
+{% set version = "0.9.9" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 6fe7830c40f20e77929b80b741a3b9f2145634b5f411176ecad0e761fef26f55
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  entry_points:
+    - orator = orator.commands.application:application.run
+
+requirements:
+  host:
+    - python =2.7|>=3.5,<3.7  # should be =2.7|>=3.5,<4 but backpack/pyaml clashes.
+    - pip
+    - poetry >=0.12a3
```

Comment:
Is `poetry` needed for building the package?

Reply:
I can't get it to build without it.